interfaces/Portalicious/src/app/components/query-table/query-table.component.html

```diff
@@ -102,26 +102,29 @@
       @for (column of columns(); track $index) {
         @if (column.field) {
           <th
-            [pSortableColumn]="
-              column.disableSorting
-                ? undefined
-                : (column.fieldForSort ?? column.field)
-            "
+            [pSortableColumn]="getColumnSortField(column)"
             style="min-width: 14rem"
           >
             <div class="flex items-center">
               <ng-container>{{ column.header }}</ng-container>
-              @if (!column.disableSorting) {
+
+              @let columnSortField = getColumnSortField(column);
+
+              @if (columnSortField) {
                 <p-sortIcon
-                  [field]="column.fieldForSort ?? column.field"
+                  [field]="columnSortField"
                   class="me-1"
                 />
               }
-              @if (!column.disableFiltering) {
+
+              @let columnFilterField = getColumnFilterField(column);
+              @let columnMatchMode = getColumnMatchMode(column);
+
+              @if (columnFilterField) {
```

Comment:
Starts to look a bit like a php-script this file.. :D

Reply:
I know, isn't it glorious? /s